### ISSUE TITLE
Loot in Geomoria - Game balance

### ISSUE THREAD

**Issue by gd2shoe** [MeseCraft/MeseCraft#75]
I just submitted a pull request.  I've written a mod that integrates Lootchests into Geomoria, basically the same way pyramids does it.
I've tried to get a reasonable game balance, but I've probably put the loot just a little too generous.  (geomoria_lootchests/loot_table.lua)

This "issue" can be ignored until someone feels like getting around to working on game balance.  I'm jotting it down here so that it doesn't get forgotten about.

**Comment by gd2shoe** [comment]
My opinion on balance:
There needs to be a reason players stop and explore Geomoria.  There simply hasn't been.  Lootchests with stuff in them can be that impetus.  
Chests aren't common, so it should feel special finding one.  They should be a bit more full than surface chests.  They should include a few things that they player can't yet make for themselves.  (Otherwise, why bother?)  But this is far too near the surface for mese shards, diamonds, etc.  (y=-170 or so)

**Comment by MeseCraft** [comment]
Yes, geomoria ended up stuck in a WIP state. I tried to restore it to be intact in the game settings, but the stalagtites and stalagmites kept breaking that (wetstone in df_caverns) kind of had me stumped. I was considering putting dwarven society around there.

**Comment by gd2shoe** [comment]
I haven't tried to fix it yet, but there are some water fountains that get broken by dynamic_liquid.  Though, dynamic_liquid does provide a solution for placing water sources that don't move, for just such an occasion.  I think it's just a matter of digging out the relevant cids, and sorting through the many room schematics looking for those fountains.

**Comment by gd2shoe** [comment]
Some actual inhabitants would do that place some good.  Do you have a mob pack in mind?
